### PR TITLE
sl-114: Site page

### PR DESCRIPTION
**Pull request by dimapopovych**
Screenshot 2023-03-29 173451
Screenshot 2023-03-29 173342
There is a little problem)

### CODE REVIEW

File: frontend/src/packages/sections/sections-api.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    siteId: number,
+  ): Promise<SectionGetAllResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(SitesApiPath.SECTIONS_BY_SITE, {
+        siteId: siteId.toString(),
+      }),
+      {
+        method: 'GET',
+        contentType: ContentType.JSON,
+        hasAuth: true,
```
Comment (dimapopovych) [inline frontend/src/packages/sections/sections-api.ts]:
```suggestion
        hasAuth: false,
```
If route will be public

File: frontend/src/pages/site/site.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  const { siteId } = useParams() as { siteId: string };
+
+  useEffect(() => {
+    void dispatch(sectionsActions.getSiteSections({ siteId: Number(siteId) }));
+  }, [dispatch, siteId]);
+
+  const renderSections = (): JSX.Element[] => {
+    return sections.map(({ type, content }) => {
+      switch (type) {
+        case 'header': {
```
Comment (dimapopovych) [inline frontend/src/pages/site/site.tsx]: SectionType.HEADER

File: frontend/src/slices/sections/actions.ts [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { type SectionGetAllItemResponseDto } from 'shared/build/index.js';
```
Comment (dimapopovych) [inline frontend/src/slices/sections/actions.ts]: import

**Comment by dimapopovych** [comment]
image

All screenshots a bit zoomed out, if it's necesarry I can record video how it looks

File: frontend/src/packages/sections/sections-api.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+class SectionsApi extends HttpApi {
+  public constructor({ baseUrl, http, storage }: Constructor) {
+    super({ path: ApiPath.SITES, baseUrl, http, storage });
+  }
+
+  public async getSectionsBySiteId(
+    siteId: number,
+  ): Promise<SectionGetAllResponseDto> {
+    const response = await this.load(
+      this.getFullEndpoint(SitesApiPath.SECTIONS_BY_SITE, {
```
Comment (what1s1ove) [inline frontend/src/packages/sections/sections-api.ts]:
```suggestion
      this.getFullEndpoint(SitesApiPath.$SITE_ID_SECTIONS, {
```

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: frontend/src/packages/sections/sections-api.ts (hunk 4 above)
Comment (what1s1ove) [reply frontend/src/packages/sections/sections-api.ts]: all dynamic parameters should start from `$`

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**